.github/workflows/test_ci.yml

```diff
@@ -50,10 +50,41 @@ jobs:
           path: artifacts
           if-no-files-found: ignore
           retention-days: 5
-      - name: Integration - dappeteer
-        if: ${{ !contains(github.event.head_commit.message, '[skip test]') }}
-        uses: mujo-code/puppeteer-headful@master
+  synpress-e2e:
+    name: Synpress e2e test
```

Comment:
any reason on creating another github action job?

Reply:
i created another github action job so that we can just retest this e2e flow when the e2e fail. but if u think its unnecessary i can revert back to just one job.

Reply:
ah ok this is fine